.github/workflows/create-release-v2.yaml

```
@@ -114,6 +114,10 @@ jobs:
     if: ${{ (always() && (contains(needs.*.result, 'success')) && !(contains(needs.*.result, 'skipped')) && !(contains(needs.*.result, 'failure')) && !(contains(needs.*.result, 'cancelled'))) }}
     name: create release and upload assets
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      # required because the tagged commit touches files under .github/workflows/
+      workflows: write
```

actionlint (1.7.12)

[error] unknown permission scope "workflows". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)